core/src/translate/function.rs

```diff
@@ -411,7 +416,11 @@ pub fn translate_function(sql_function: &SqlFunction) -> Result<Expr> {
             check_len(name, args.len(), 2)?;
 
             let expr = translate_expr(args[0])?;
-            let format = translate_expr(args[1])?;
+            let format = match args_str[1].as_str() {
```

Comment:
If you use only the first index item, it seems better to replace `args_str` with a single item.
ex)
```rust
let args = function_args_exprs
     .iter()
     .map(ToString::to_string)
     .skip(1)
     .next();
```